language_negotiation_lambda.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
Should this all be wrapped in a function so that this doesn't affect anything else?